app/celery/provider_tasks.py

```diff
@@ -128,6 +128,8 @@ def deliver_sms(self, notification_id):
             )
         # Code branches off to send_to_providers.py
         message_id = send_to_providers.send_sms_to_provider(notification)
+
+        # DEPRECATED
```

Comment:
What is this deprecated note for?  Indicating that the code below will go away and just a heads up, or something else?

Reply:
It is the call to the one-at-at-time delivery receipts which will be removed when #1465 is done.